src/trsp/GraphDefinition.cpp

```diff
@@ -626,11 +626,11 @@ bool GraphDefinition::addEdge(edge_t edgeIn) {
     if (itNodeMap != m_mapNodeId2Edge.end()) {
         // Connect current edge with existing edge with start node
         // connectEdge(
-        int64 lEdgeCount = itNodeMap->second.size();
+        int64 lEdgeCount = static_cast<int64>(itNodeMap->second.size());
```

Comment:
as in other cases, I feel that this is better declared with `auto`

Reply:
Not changing more than needed on trsp